data/json/npcs/missiondef.json

```diff
@@ -971,6 +971,75 @@
       "success_lie": "What good does this do us?",
       "failure": "It was a lost cause anyways..."
     }
+  },
+    {
+    "id": "MISSION_RANCH_CHOP_SHOP_1",
+    "type": "mission_definition",
+    "name": "Gather 1200L of Gasoline",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 5,
+    "value": 50000,
+    "item": "gasoline",
+    "count": 1200000,
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_RANCH_CHOP_SHOP_2",
+    "dialogue": {
+      "describe": "We need help...",
+      "offer": "Getting spare parts for vehicles is never really an issue.  Unfortunately the same can’t be said for gasoline.  Sending out people to siphon fuel from vehicles or gas stations is a dangerous job.  It’s even worse if people driving have to stop to fill up.  Ideally we’d like to send people out with a full tank of fuel so they don’t have to make any dangerous detours.  I’d like us to have our own little service station for refueling but for that we’re going to need at least 1200L of gasoline.  I don’t have anyone crazy enough on hand to attempt that sadly, if you think you can pull it off the job is yours.",
+      "accepted": "I'm counting on you.",
+      "rejected": "Come back when you get a chance, we need skilled survivors.",
+      "advice": "Finding gasoline is never an issue, there’s plenty of cars still filled in the city and plenty of gas stations.  It’s the time to siphon it and transport it that’s the problem.  To make it easier you might want to see about transporting it in a truck or a tanker.",
+      "inquire": "Do you have the gasoline we need?",
+      "success": "Thank you for your assistance.",
+      "success_lie": "What good does this do us?",
+      "failure": "It was a lost cause anyways..."
+    }
+  },
+  {
+    "id": "MISSION_RANCH_CHOP_SHOP_2",
+    "type": "mission_definition",
+    "name": "Find a book detailing how to make Biodiesel",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 5,
+    "value": 50000,
+    "item": "textbook_biodiesel",
+    "start": "ranch_chop_shop_1",
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_RANCH_CHOP_SHOP_3",
+    "dialogue": {
+      "describe": "We need help...",
+      "offer": "Getting cars fueled up for trips has been a lot less hassle with our own supply of gasoline to fill vehicles.  As great as that is though I have some concerns about keeping things running in the future.  I think we seriously need to look into creating our own fuel source.  With that in mind I’ve been working on some diesel engines that came through the garage to get them into working order.  With the right equipment it’s possible to make biodiesel unfortunately I don’t have a book on hand to tell me how to do it.  If you can manage to find a book detailing the process to making diesel it would be a great help.",
```

Comment:
```suggestion
      "offer": "Getting cars fueled up for trips has been a lot less hassle with our own supply of gasoline to fill vehicles.  As great as that is though I have some concerns about keeping things running in the future.  I think we seriously need to look into creating our own fuel source.  With that in mind I've been working on some diesel engines that came through the garage to get them into working order.  With the right equipment it's possible to make biodiesel unfortunately I don't have a book on hand to tell me how to do it.  If you can manage to find a book detailing the process to making diesel it would be a great help.",
```